README.yaml

```diff
@@ -57,9 +63,10 @@ references:
 
 # Short description of this project
 description: |-
-  This is `terraform-example-module` project provides all the scaffolding for a typical well-built Cloud Posse module. It's a template repository you can
-  use when creating new repositories.
+  This is `terraform-aws-msk-apache-kafka-cluster` to provision AWS MSK.
```

Comment:
Terraform module to provision [Amazon Managed Streaming for Apache Kafka](https://aws.amazon.com/msk/) cluster

Reply:
@aknysh added